llvm/lib/Transforms/Scalar/JumpThreading.cpp

```diff
@@ -2523,14 +2523,9 @@ bool JumpThreadingPass::doesBlockHaveProfileData(BasicBlock *BB) {
   const Instruction *TI = BB->getTerminator();
   assert(TI->getNumSuccessors() > 1 && "not a split");
 
-  MDNode *WeightsNode = TI->getMetadata(LLVMContext::MD_prof);
+  auto *WeightsNode = getBranchWeightMDNode(*TI);
   if (!WeightsNode)
     return false;
-
```

Comment:
ultra nit: maybe keep one of the newlines

Reply:
The phab revision changed this further.